frontend/src/components/common/ShowHideLabel/ShowHideLabel.tsx

```diff
@@ -43,7 +43,7 @@ export default function ShowHideLabel(props: ShowHideLabelProps) {
     <Box display={expanded ? 'block' : 'flex'}>
       <label
         id={labelIdOrRandom}
-        style={{ wordBreak: 'break-all', whiteSpace: 'normal' }}
+        style={{ wordBreak: 'normal', whiteSpace: 'normal' }}
```

Comment:
Is the break-all change actually causing a regression? (Do you have a screenshot?)

Reply:
yeah you're right it's not, I've removed it. It was just really hard to read error messages in the cluster overview and I thought it must be a regression. We can come back to this later